app/javascript/spec/workers-form/workers-form.spec.js

```diff
@@ -48,17 +48,15 @@ describe('Workers form', () => {
                 memory_threshold: 2147483648,
               },
             },
-            defaults: {
-              memory_threshold: '500.megabytes',
-            },
+            defaults: {},
```

Comment:
I removed the 500.megabytes example for the queue_worker_base defaults to recreate the situation we have on master where some queue based workers have a value specified and others inherit from worker_base